packages/fiori-app-sub-generator/test/unit/fiori-app-generator/fioriAppGenerator-lifecycle1.test.ts

```diff
@@ -406,90 +407,116 @@ describe('Test FioriAppGenerator', () => {
         });
     });
 
-    test('Should prompt for entity related answers custom page ', async () => {
-        // Force cache usage, YUI only
-        (getHostEnvironment as jest.Mock).mockReturnValue(hostEnvironment.vscode);
-        // Must be a datasource that provides an service to be cached (DatasourceType.none is not cached)
-        odataServiceAnswers.source = DatasourceType.odataServiceUrl;
-        // Set cache to simulate back navigation state (edmx and cap service) restoration entity prompting
-        const mockCachedService: Service = {
-            edmx: '<edmx></edmx>',
-            capService: {
-                projectPath: '/cap/project/path',
-                serviceName: 'aCapService'
-            },
-            annotations: [
-                {
-                    TechnicalName: 'testAnnotationsTechName',
-                    Definitions: 'testAnnotations',
-                    Version: '1.0.0',
-                    Uri: 'testUri'
-                }
-            ],
-            source: DatasourceType.capProject
-        };
-        // Note: These are not real-world combinations of options and state, they are for testing as many branches and options as possible
-        (options.appWizard as AppWizardCache)!['$fiori-cache'] = {
-            service: mockCachedService
+    describe('Test FioriAppGenerator for FPM projects', () => {
```

Comment:
Why was the existing test replaced? Can you please restore it and add new tests for your specific use case.